src/engine/source/kvdb/src/kvdb.cpp

```diff
@@ -235,7 +233,8 @@ struct KVDB::Impl
             m_CFDescriptors.erase(std::remove_if(m_CFDescriptors.begin(),
                                                  m_CFDescriptors.end(),
                                                  [&](auto const& des)
-                                                 { return des.name == columnName; }));
+                                                 { return des.name == columnName; }),
+                                  m_CFDescriptors.end());
```

Comment:
this call will remove at most one item even when multiple items should be removed, this is a idiom of the C++ STL < std20

 https://en.wikipedia.org/wiki/Erase%E2%80%93remove_idiom